be/src/common/config.h

```diff
@@ -966,6 +966,13 @@ CONF_mInt64(l0_max_mem_usage, "67108864");  // 64MB
 CONF_mInt64(l0_snapshot_size, "16777216"); // 16MB
 CONF_mInt64(max_tmp_l1_num, "10");
 CONF_mBool(enable_parallel_get_and_bf, "true");
+CONF_mBool(enable_pindex_filter, "true");
```

Comment:
```enable_parallel_get_and_bf``` and ```enable_pindex_filter``` are redundant?

Reply:
> `enable_parallel_get_and_bf` and `enable_pindex_filter` are redundant?

Actually `enable_parallel_get_and_bf` should be replaced with `enable_parallel_get` because this configure doesn't affect bloom filter any more. Keep the origin configure just for compatibility.

Reply:
> How many users have used this `enable_parallel_get_and_bf` configuration? If only a little, you can use `enable_pindex_filter` instead.

No, these two configures are different. `enable_parallel_get_and_bf` has nothing to do with bloom filter after this pr and it determines whether or not to parallelize query when we get from persistent index. And `enable_pindex_filter ` is determines to whether or not to use bloom filter. So i think these two configures can not replace each other.